go_test.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import unittest
+import time
+
+import go
+
+
+class GeneralTestCases(unittest.TestCase):
+    def test_deampify_url(self):
+        input_string = 'https://www.example.com/webhp?sourceid=chrome-instant&amp;ion=1&amp;espv=2&amp;ie=UTF-8'
+        expected = 'https://www.example.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8'
+        self.assertEqual(expected, go.deampify(input_string))
+
+    def test_prettyday_should_return_never(self):
+        self.assertEqual('never', go.prettyday(0))
+        self.assertEqual('never', go.prettyday(-1))
+
+    def test_prettyday_should_return_today(self):
+        _today = go.today()
+        self.assertEqual('today', go.prettyday(_today))
+
+    def test_prettyday_should_return_yesterday(self):
+        yesterday = go.today() - 1
+        self.assertEqual('yesterday', go.prettyday(yesterday))
+
+    def test_prettyday_should_return_num_of_days(self):
+        today = go.today()
+        month_ago = today - 30
+        self.assertEqual('30 days ago', go.prettyday(month_ago))
+
+    def test_prettyday_should_return_num_of_months(self):
+        today = go.today()
+        months_ago = today - 95
+        self.assertEqual('3 months ago', go.prettyday(months_ago))
+
+    def test_prettytime_should_return_never(self):
+        self.assertEqual('never', go.prettytime(420))
+        self.assertEqual('never', go.prettytime(-1))
+
+    def test_prettytime_should_return_today(self):
+        timestamp = time.time()
+        self.assertEqual('today', go.prettytime(timestamp))
+
+    def test_prettytime_should_return_yesterday(self):
+        timestamp = time.time() - (24 * 3600)
+        self.assertEqual('yesterday', go.prettytime(timestamp))
+
+    def test_prettytime_should_return_num_of_days(self):
+        timestamp = time.time() - (6 * 24 * 3600)
+        self.assertEqual('6 days ago', go.prettytime(timestamp))
+
+    def test_prettytime_should_return_num_of_months(self):
+        timestamp = time.time() - (95 * 24 * 3600)
+        self.assertEqual('3 months ago', go.prettytime(timestamp))
+
+    def test_should_return_true_for_int(self):
+        self.assertTrue(go.is_int(5))
+
+    def test_should_return_false_for_int(self):
+        self.assertFalse(go.is_int('foo'))
+
+    def test_makeList_should_return_list(self):
+        _list = [1, 2, 3]
+        _num_set = {42, 35}
```

Comment:
For now I'll change the test to support it as I don't know of anything inherently broke about it though it would be old, in the future I'll probably add the 3.x to the test path and remove 2.6, but given that I haven't done any tests around it I'm a bit wary of including it at the moment and will work on that when i have time.